src/OrderFormAcceptanceTests.Actions/Pages/OrganisationsOrdersDashboard.cs

```diff
@@ -16,14 +16,14 @@ public OrganisationsOrdersDashboard(IWebDriver driver) : base(driver)
         public void WaitForDashboardToBeDisplayed()
         {
             Driver.WaitForJsToComplete(Wait);
-            Wait.Until(d => d.FindElements(Pages.OrganisationsOrdersDashboard.PageTitle).Count == 1);
-            Wait.Until(d => CreateNewOrderButtonDisplayed());
+            //Wait.Until(d => d.FindElements(Pages.OrganisationsOrdersDashboard.PageTitle).Count == 1);
+            //Wait.Until(d => CreateNewOrderButtonDisplayed());
         }
 
         public void SelectExistingOrder(string callOffAgreementId)
         {
             Wait.Until(d => d.FindElements(Pages.OrganisationsOrdersDashboard.SpecificExistingOrder(callOffAgreementId)).Count == 1);
-            Wait.Until(ElementExtensions.ElementToBeClickable(Pages.OrganisationsOrdersDashboard.SpecificExistingOrder(callOffAgreementId)));
+            //Wait.Until(ElementExtensions.ElementToBeClickable(Pages.OrganisationsOrdersDashboard.SpecificExistingOrder(callOffAgreementId)));
```

Comment:
What's your plan with the commented-out lines?

Reply:
I've pushed a commit that should have uncommented them